VAMobile/src/screens/HomeScreen/HomeScreen.test.tsx

```diff
@@ -52,4 +68,38 @@ context('HomeScreen', () => {
       expect(mockNavigationSpy).toBeCalledWith('Webview', { displayTitle: 'va.gov', url: 'https://www.va.gov/find-locations/', loadingMessage: 'Loading VA location finder...' })
     })
   })
+
+  // it('displays prescriptions module when there are active prescriptions', () => {
+  //   initializeTestInstance(2)
+  //   expect(screen.getByText('Prescriptions')).toBeTruthy()
+  //   expect(screen.getByText('(2 active)')).toBeTruthy()
+  // })
+
+  // it('navigates to prescriptions screen when prescriptions module is tapped', () => {
+  //   initializeTestInstance(2)
+  //   fireEvent.press(screen.getByText('Prescriptions'))
+  //   expect(Linking.openURL).toBeCalledWith('vamobile://prescriptions')
+  // })
+
+  // it('does not display prescriptions module when there are no active prescriptions', () => {
+  //   initializeTestInstance(0)
+  //   expect(screen.queryByText('Prescriptions')).toBeFalsy()
+  // })
+
+  // it('displays claims module when there are active claims', () => {
+  //   initializeTestInstance(0, 2)
+  //   expect(screen.getByText('Claims')).toBeTruthy()
+  //   expect(screen.getByText('(2 open)')).toBeTruthy()
+  // })
+
+  // it('navigates to claims history screen when claims module is tapped', () => {
+  //   initializeTestInstance(0, 2)
+  //   fireEvent.press(screen.getByText('Claims'))
+  //   expect(Linking.openURL).toBeCalledWith('vamobile://claims')
+  // })
+
+  // it('does not display claims module when there are no active claims', () => {
+  //   initializeTestInstance(0)
+  //   expect(screen.queryByText('Claims')).toBeFalsy()
+  // })
```

Comment:
This can be removed from the PR. These tests may even change in the feature-xl branch